akka-discovery/src/main/resources/reference.conf

```diff
@@ -61,8 +61,8 @@ akka.discovery {
   aggregate {
     class = akka.discovery.aggregate.AggregateServiceDiscovery
 
-    # List of service discovery methods to try in order. E.g DNS then fall back to config
-    # ["akka-dns" , "confg" ]
+    # List of service discovery methods to try in order. E.g config then fall back to DNS
+    # ["config", "akka-dns"]
```

Comment:
reference.conf is often used as documentation and best-practice reference. I find it useful to promote the idea to first lookup on a static configuration and then fall back to a method requiring remote calls. 

Moreover, there was a typo in `confg` (nitpicking).